client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -146,6 +146,100 @@ def download_model(self, download_to_path):
         """
         raise NotImplementedError
 
+    @staticmethod
+    def _build_artifact_store_path(artifact_stream, key, ext=None):
+        """Build a path for Verta's backend artifact store to store `artifact_stream`.
+
+        Parameters
+        ----------
+        artifact_stream : file-like
+            Artifact bytes.
+        key : str
+            Artifact key.
+        ext : str, optional
+            File extension associated with `artifact_stream`.
+
+        Returns
+        -------
+        str
+            Artifact store path for `artifact_stream`
+
+        """
+        # calculate checksum
+        checksum = _artifact_utils.calc_sha256(artifact_stream)
+        artifact_stream.seek(0)
```

Comment:
Is this necessary? In the code for `calc_sha256` I see:
```
    finally:
        reset_stream(bytestream)  # reset cursor to beginning as a courtesy
```

Reply:
Good point! I'll remove it for simplicity's sake